src/main/drivers/pwm_output.h

```diff
@@ -29,6 +29,26 @@
 #endif
 
 typedef enum {
+    DSHOT_CMD_MOTOR_STOP = 0,
+    DSHOT_CMD_BEEP1,
+    DSHOT_CMD_BEEP2,
+    DSHOT_CMD_BEEP3,
+    DSHOT_CMD_BEEP4,
+    DSHOT_CMD_BEEP5,
+    DSHOT_CMD_ESC_INFO,
+    DSHOT_CMD_SPIN_ONE_WAY, 
+    DSHOT_CMD_SPIN_OTHER_WAY, 
+    DSHOT_CMD_3D_MODE_OFF,
+    DSHOT_CMD_3D_MODE_ON, 
+    DSHOT_CMD_SETTINGS_REQUEST,
+    DSHOT_CMD_SAVE_SETTINGS, 
+    DSHOT_CMD_ROTATE_NORMAL = 20, //Blheli_S only command
+    DSHOT_CMD_ROTATE_REVERSE = 21,  //Blheli_S only command
```

Comment:
Does the "DSHOT_CMD_SPIN_OTHER_WAY" always make kiss escs spin the opposite way as they are configured (CW ->CCW and CCW->CW)?   If so it would be easy to standardize, if not there is no way to tell what way the motor is spinning without telemetry or telling betaflight what each motor is doing ahead of time.